kubeflow/core/update_backend.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# A simple shell script to configure the backend timeouts and health checks by using gcloud.
+[ -z ${NAMESPACE} ] && echo Error NAMESPACE must be set && exit 1
```

Comment:
As noted in my comment about using a stateful set vs. K8s job,

should we add a while loop around this code and repeat the activation every hour or so?

Reply:
I added a `sleep 3600` at the end of the script, after the container completes, k8s should spin up another pod